src/cascadia/TerminalSettingsEditor/Microsoft.Terminal.Settings.Editor.vcxproj

```diff
@@ -207,6 +217,10 @@
     <PRIResource Include="Resources\en-US\Resources.resw">
       <SubType>Designer</SubType>
     </PRIResource>
+    <!-- Make sure to include the CommandlineArgs resources here, so they show up in our scope -->
+    <PRIResource Include="$(OpenConsoleDir)src\cascadia\CommandlineArgs\Resources\en-US\Resources.resw">
+      <SubType>Designer</SubType>
+    </PRIResource>
     <OCResourceDirectory Include="Resources" />

```

Comment:
you'll need to add a recursive resource directory using OCResourceDirectory that points to CommandlineArgs, otherwise we will only ever get the english ones. OCRes is the way that the language-named directories get pulled in. :smile: